lib/rest2.js

```diff
@@ -256,11 +256,11 @@ class RESTv2 {
 
   /**
    * @param {Array[]} data order matters
-   * @return {Array[]} merged arr of currencies and names sorted with no pairs repeated
+   * @return {Array[]} merged arr of currencies and names sorted with no pairs repeated adding pool and explorer to each
    * @private
    */
-  _mergeCurrenciesListSymLabel (data) {
-    if (!Array.isArray(data) || data.length !== 3) {
+  _mergeCurrenciesListSymLabelAddPoolAndExplorer (data) {
```

Comment:
Can you rename this to `_genCurrencyList` so it's shorter? No need to describe everything it does in the function name, the doc header is for that :)

Reply:
yes, on it